apps/sparo-lib/assets/sparo-profile.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Use this file instead:

https://github.com/tiktok/sparo/blob/main/apps/website/docs/pages/configs/profile_json.md